elastic-search-support/src/main/resources/application.yml

```diff
@@ -91,6 +91,7 @@ elasticsearch:
     created_date: ${elasticsearch.elasticMappings.defaultDate}
     last_modified: ${elasticsearch.elasticMappings.defaultDate}
     security_classification: ${elasticsearch.elasticMappings.defaultKeyword}
+    case_access_category:   ${elasticsearch.elasticMappings.defaultText}
```

Comment:
```suggestion
```
We should be able to delete this as these properties won't be getting used for the global search index

Reply:
DONE